File: 0014-longest-common-prefix/0014-longest-common-prefix.java
class Solution {
    public String longestCommonPrefix(String[] strs) {
        String Prefix = strs[0];
        for(int i =0;i<strs.length;i++){
            while(strs[i].indexOf(Prefix)!=0){
                Prefix = Prefix.substring(0,Prefix.length()-1);
                if(Prefix.length() == 0) return "";
            }
        }
    return Prefix;}
}

File: 0014-longest-common-prefix/0014-longest-common-prefix.js
/**
 * @param {string[]} strs
 * @return {string}
 */
var longestCommonPrefix = function (strs) {
    strs.sort();
    var ans = ""
    if (strs[0] == "") {
        return ans;
    }
    else {
        for (let i = 0; i < strs[0].length; i++) {
            if(strs[0][0]!=strs[strs.length-1][0]){
                break;
            }
            if (strs[0][i] === strs[strs.length - 1][i]) {
                ans += strs[0][i];

            }
        }
    }
    return ans;
};